doc/src/create_atoms.rst

```diff
@@ -28,11 +28,13 @@ Syntax
 * zero or more keyword/value pairs may be appended
 * keyword = *mol* or *basis* or *ratio* or *subset* or *remap* or *var* or *set* or *rotate* or *units*
 
-  .. code-block:: LAMMPS
+  .. parsed-literal::
 
-       *mol* value = template-ID seed
+       *mol* value = template-ID seed [*index* molindex]
```

Comment:
I think this should be enabled by a separate, optional molfrac keyword which allows specification of probabilities for multiple molecules in the template.  Same as fix deposit and fix pour.  And don't change the syntax of the mol keyword.

Reply:
this if fully backward compatible

Reply:
i see the point of wanting to be consistent with fix pour and fix deposit, though.

Reply:
this change has been reverted and the implementation of a "molfrac" version is postponed until after the stable release.